outlookmail/tests/test.bal

```diff
@@ -0,0 +1,414 @@
+import ballerina/log;
+import ballerina/os;
+import ballerina/test;
+import ballerina/io;
+
+configurable string refreshUrl = os:getEnv("REFRESH_URL");
+configurable string refreshToken = os:getEnv("REFRESH_TOKEN");
+configurable string clientId = os:getEnv("CLIENT_ID");
+configurable string clientSecret = os:getEnv("CLIENT_SECRET");
+
+Configuration configuration = {
+    clientConfig: {
+        refreshUrl: refreshUrl,
+        refreshToken: refreshToken,
+        clientId: clientId,
+        clientSecret: clientSecret
+    }
+};
+
+Client oneDriveClient = check new (configuration);
+string messageId = "";
+string createdDraftId = "";
+string sentMessageId = "";
+string mailFolderId = "";
+string searchMailFolderId = "";
+string attachmentId = "";
```

Comment:
There is already a constant called `EMPTY_STRING`
Shall we use that instead of using `""` everywhere